catkin_ws/src/acr_debug/src/keyboard_sensors/keyboard_sensors.py

```diff
@@ -95,80 +124,60 @@ def getKey():
 	rospy.init_node('acr_keyboard')
 
 	try:
-		print msg
-		poi = float('nan')
-
-		ult_max = 10000
-		ult_dist = 1999
+		ult_max = 500
+		ult_dist = 100
```

Comment:
You should also update the min distance attribute in sensordata.hpp. This attribute is set to 2000, causing the SensorData class to always think there is something right in front of the robot.